fabfile.py

```diff
@@ -13,13 +13,41 @@
 CURRENT_DIR = os.path.dirname(__file__)
 
 
+TIERS = {
+    'production': {
+        'hosts': ['root@104.131.92.59'],
```

Comment:
I don't think there would be a way around that besides relying on people having the correct ssh config set up. Which removes it from version control, which is not good.